colossalai/engine/gradient_accumulation/__init__.py

```diff
@@ -6,6 +6,11 @@
 from torch.optim.lr_scheduler import _LRScheduler
 from ._gradient_accumulation import GradAccumDataloader, GradAccumOptimizer, GradAccumLrSchedulerByStep, GradAccumGradientHandler
```

Comment:
There is a test for gradient accumulation in `test_utils`. I guess this movement will lead to test error.

Reply:
OK, I will fix it